src/content/docs/en/guides/internationalization.mdx

```diff
@@ -0,0 +1,325 @@
+          - about.astro
+          - index.astro
+        - es
+          - about.astro
+          - index.astro
+        - pt_BR
+          - about.astro
+          - index.astro
+        - index.astro  (redirects to the route created for `/en/index.astro`)
+    </FileTree>
+
+    :::note
+    The localized folders do not need to be at the root of the `/pages/` folder.
+    
+    Create individual `/[lang]/` folders anywhere within `src/pages/` and Astro's [file-based routing](/en/core-concepts/routing/) will create your pages at corresponding URL paths. 
+    
+    ```
+    src/pages/blog/[lang]/about.astro -> example.com/blog/es/about/
+    ```
+    :::
+
+3. When writing links to pages on your site, use a localized URL. You can write this URL directly, or compute it using the [`getRelativeLocaleURL()`](#getrelativelocaleurl) helper available from the [`astro:i18n` module](#virtual-module-astroi18n). This will always provide the correct, localized route and can help you correctly use, or check, URLs on your site.
+
+    ```astro title="src/pages/es/index.astro"
+    ---
+    import { getRelativeLocaleUrl } from 'astro:i18n';
+    const aboutURL=getRelativeLocaleUrl("es", "about");
+    ---
+
+    <a href="/es/get-started/">¡Vamos!</a>
+    <a href={getRelativeLocaleUrl('es', 'blog')}>Blog</a>
+    <a href={aboutURL}>Acerca</a> 
+    ```
+
+### `defaultLocale` and `locales`
+
+Both a default language ([`defaultLocale`](en/reference/configuration-reference/#experimentali18ndefaultlocale)) and a list of all supported languages ([`locales`](en/reference/configuration-reference/#experimentali18nlocales)) must be specified in your `i18n` routing configuration.
+
+Each language must be a string (e.g. `"fr"`, `"pt_BR"`), but no particular language format or syntax is enforced while this feature is still experimental and under development. This may be subject to change in future versions.
+
+Your `/[lang]/` folder names must match exactly the `locales` in the list, and every supported language must have its own localized folder. 
+
+Depending on your deploy host, you may discover transformations in URL paths, so check your deployed site to determine the best syntax for your project.
+
+### Routing strategy 
+
+You can choose whether or not pages in your `defaultLocale` should be prefixed with their `/lang/` in their URL path. Every other supported locale will automatically be prefixed.
+
+#### `'prefix-other-locales'`
+
+```js title="astro.config.mjs" ins={7}
+import { defineConfig } from "astro/config"
+export default defineConfig({
+  experimental: {
+    i18n: {
+      defaultLocale: "en",
+      locales: ["es", "en", "fr"],
+      routingStrategy: "prefix-other-locales"
+    }
+  }
+})
+```
+
+This is the **default** value. With this option, URLs in your default language will **not** have a `/[lang]/` prefix.
+
+For example, if `i18n.defaultLocale: "en"` is configured:
+
+- `src/pages/en/blog.astro` will produce the route `example.com/blog/` 
```

Comment:
This isn't correct. With the `'prefix-other-locales'`, developers must have `src/pages/blog.astro` in their file system, because it's "assumed" that pages outside of a lang/locale folder belong to the default locale. This is related to my comment above.